nxstacker/schema/nxstackerSchema.json

```diff
@@ -0,0 +1,283 @@
+{
+  "type": "object",
+  "properties": {
+    "experimentType": {
```

Comment:
The name of the parameter, here `experimentType` has to match the name that is used in the list of arguments in the template, currently `experiment-type` (and obviously has to also match the reference in the UI schema which is currently `experimentType`.

To fix this, there are two options, either change all the parameter names in the schema and UI schema to match the template or the other way around, i.e. change the names in the workflow template (be aware that they are used in multiple places). Both options are equally valid.

Reply:
Thank you for the note here and reminder. I did speak with Victoria about the schema naming convention beforehand. I think camelCase was suggested as being more logical to use in the JSON schema. I have seen snake_case and hyphen-case used in similar scenarios though.

I gave it a go updating the workflow template yaml file with the camelCase. ([commit](https://github.com/DiamondLightSource/imaging-workflows/pull/21/commits/4760493e523c678024de488a54966a7ded817691))

If a different convention makes sense with this application, or with consideration to other diamond applications, I'm happy to alter.